src/main/java/com/redhat/cloud/notifications/db/BundleResources.java

```diff
@@ -0,0 +1,143 @@
+package com.redhat.cloud.notifications.db;
+
+import com.redhat.cloud.notifications.models.Application;
+import com.redhat.cloud.notifications.models.Bundle;
+import io.r2dbc.postgresql.api.PostgresqlResult;
+import io.smallrye.mutiny.Multi;
+import io.smallrye.mutiny.Uni;
+import reactor.core.publisher.Flux;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Deal with Bundles.
+ * More or less stupid copy & paste & search & replace of
+ * code in BundleResources
```

Comment:
Did you do search & replace on this comment too? :grin:  (Was expecting to see `code in ApplicationResources`)

Reply:
I guess I should change this a bit :)
I startet with c&p&s&r, but then also introduced that common base class (for deletes atm only), which could probably also be used elsewhere